pelican/themes/simple/templates/article.html

```diff
@@ -22,11 +22,10 @@
 {% endblock %}
 
 {% block content %}
-<section id="content" class="body">
   <header>
-    <h2 class="entry-title">
+    <h1 class="entry-title">
```

Comment:
If we're moving simple theme to a classless theme we can remove also this class and all the others.

Reply:
I decided to leave this class because in the `base.html` also has an `<h1>` tag within a `<header>` tag, so I thought it would still be easier to target the entry title using a dedicated class in CSS. What do you think?